ios/FluentUI/Tab Bar/TabBarItem.swift

```diff
@@ -90,6 +114,19 @@ open class TabBarItem: NSObject {
     /// Notification sent when the tab bar item's badge value changes.
     static let badgeValueDidChangeNotification = NSNotification.Name(rawValue: "TabBarItemBadgeValueDidChangeNotification")
 
+    /// Notification sent when item's `isUnread` value changes.
+    static let isUnreadValueDidChangeNotification = NSNotification.Name(rawValue: "TabBarItemisUnreadValueDidChangeNotification")
+
+    /// This value will determine whether or not to show the mark that represents the "unread" state.
```

Comment:
the documentation and the code should be clear which will win. BadgeValue? or isUnread?